src/application/modules/legacy/module.ts

```diff
@@ -60,7 +60,7 @@ export class LegacyModule extends BaseModule {
 
 	public constructor() {
 		super();
-		this.stores.register(LegacyAccountStore, new LegacyAccountStore(this.name));
+		this.stores.register(LegacyAccountStore, new LegacyAccountStore(this.name, 0));
 		this.events.register(ReclaimLSKEvent, new ReclaimLSKEvent(this.name));
```

Comment:
Could we please add an empty line for visual clarity in separation of store and event registration.
```suggestion
		this.stores.register(LegacyAccountStore, new LegacyAccountStore(this.name, 0));

		this.events.register(ReclaimLSKEvent, new ReclaimLSKEvent(this.name));
```

Reply:
Also, seeing the following error when running the node with the chain connector enabled:

Start command:
```
./bin/run start -n devnet --api-ipc -l debug --api-ws --enable-chain-connector-plugin
```

Logged error:
```
2023-02-27T09:16:56.764Z DEBUG Sameers-MacBook-Pro.local plugin_chainConnector 27584 [plugin=chainConnector] Plugin is registered to bus
2023-02-27T09:16:56.767Z ERROR Sameers-MacBook-Pro.local plugin_chainConnector 27584 [err=Lisk validator found 3 error[s]:
Missing property, must have required property 'ccuFee'
Missing property, must have required property 'encryptedPrivateKey'
Missing property, must have required property 'password'] Fail to handle message.
```

Reply:
I think all of these are optional config params? @shuse2 is this correct?